Xamarin.Forms.Core.UnitTests/ButtonUnitTest.cs

```diff
@@ -316,6 +316,14 @@ public void ButtonBorderRadiusSetMinusOne()
 			Assert.AreEqual(-1, button.BorderRadius);
 		}
 
+		[Test]
+		public void SetLetterSpacing()
+		{
+			var control = new Button();
+			control.PropertyChanged += (sender, args) => Assert.AreEqual(nameof(ITextElement.LetterSpacing), args.PropertyName);
+			control.LetterSpacing = 1.0;
+		}
```

Comment:
I'm not sure about testing a property change event. It would be more useful to have a special test page in `ControlGallery` for manual visual testing.

Reply:
I think that this can be left in order not to forget that there are properties. and I will add a page with UI elements

Reply:
I added the "Letter Spacing Views Gallery" page